thunder/tests/test_ops.py

```diff
@@ -251,3 +251,32 @@ def fn(a):
     out = jf(a)
     assert_close(a, a_ref)
     assert_close(out, out_ref)
+
+def test_exponential():
+    def fn(a):
+        return a.exponential_(1)
+
+    size = 1000
+    seed = 1234
+    with torch.device("cuda"):
+        a_ref = torch.ones(size)
+        torch.manual_seed(seed)
+        a_ref = fn(a_ref)
+
+        a = torch.ones(size)
+        torch.manual_seed(seed)
+
+        # nbfuser fuses prims.uniform, which is used by exponential resulting in differing numerics.
+        executors = thunder.get_default_executors()
+        fuser_i = None
+        for i, e in enumerate(executors):
+            if e.name == "nvfuser":
+                fuser_i = i
+                break
+
+        executors = executors[:fuser_i]+ executors[fuser_i+1:] if fuser_i is not None else executors
+        jf = thunder.jit(fn, executors=executors)
```

Comment:
```suggestion
        # nvfuser fuses prims.uniform and implements it slightly differently to PyTorch, leading to different random samples
        jf = thunder.jit(fn, executors=())
```

I think we can have it easy here.